src/app/clusters/time-synchronization-server/time-synchronization-delegate.h

```diff
@@ -0,0 +1,57 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app-common/zap-generated/attributes/Accessors.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace TimeSynchronization {
+
+using namespace chip::app::Clusters::TimeSynchronization;
+using TimeZoneList = DataModel::List<chip::app::Clusters::TimeSynchronization::Structs::TimeZoneStruct::Type>;
```

Comment:
I do really need Accessors.h and why to add the others when already available?

Reply:
Because you don't know that Accessors.h will keep including them.  You should include any types that you are directly referencing in this header, so you don't rely on the structure of other headers.